frontend/simple/views/components/Modal/ModalHeader.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <header class="modal-card-head has-text-centered">
+    <h1 class="modal-card-title title is-size-5 is-marginless has-text-grey" v-if="$slots.subTitle">
+      <slot></slot>
+    </h1>
+    <h2 class="title is-size-3">
+      <slot name="subTitle" v-if="$slots.subTitle"></slot>
+      <slot v-else></slot>
```

Comment:
This should be a named slot too..?

Reply:
Most of the use cases you will need only a title, so I left that as the default slot.

```vue
<!-- With default slot - simplest solution -->
<modal-header>
  Heading
<modal-header>

<!-- With only named slots - meh -->
<modal-header>
  <template slot="title">Heading</template>
<modal-header>